server/src/controller/authControllers.ts

```diff
@@ -155,9 +156,74 @@ const login = async (req: Request, res: Response) => {
   }
 };
 
+const sendPasswordResetEmail = async (req: Request, res: Response) => {
+  try {
+    const {email}=req.body;
+    
+    const result = await db.pool.query(
+      "SELECT email FROM public.users WHERE email = $1",
+      [email]
+    );
+   
+    if(result.rowCount==0){
+      res.json({status:false,error:false});
```

Comment:
add proper status code and also add message instead of status or error.

Reply:
Okay